runner/config.go

```diff
@@ -37,6 +37,7 @@ type cfgBuild struct {
 	IncludeDir       []string      `toml:"include_dir"`
 	ExcludeFile      []string      `toml:"exclude_file"`
 	ExcludeUnchanged bool          `toml:"exclude_unchanged"`
+	FollowSymlink    bool          `toml:"follow_symlink"`
```

Comment:
It's better to add usage in `air_example.toml`.

Reply:
It has been updated.